### PR TITLE
WIP: Feature/sqlite migration support take2

### PR DESCRIPTION
**Pull request by JC008**
- implemeted ISQLiteDatabaseMigrator
- converted inline sql to embedded resources
- implemented Helpz.SQLite in tests

### CODE REVIEW

**Comment by rasmus** [comment]
@JC008 It seems like I messed up the webhooks integration for pull requests. They should be working now and I'll test it tonight if I get the time.

**Comment by rasmus** [comment]
@JC008 Build integration has been fixed

**Comment by rasmus** [comment]
@JC008 Let me know if this is finished

**Comment by JC008** [comment]
Doing a merge of latest develop now.

**Comment by JC008** [comment]
Thats looks like a regression on the Moq and AutoFixture - Sorry! Can you discard ddb4c1f?

**Comment by rasmus** [comment]
@JC008 The commit dca15f4cf2de71dcc10f1e7973e383a05c4b56e4 didn't have any changes to the `paket.lock` file, so you might have forgotten to check that in when you did the update. The DLL referenced between the two versions changes from `net40 to `net45`

**Comment by rasmus** [comment]
@JC008 The build is still failing though. Try doing a `.paket/paket.exe install` and check the updated files

**Comment by JC008** [comment]
@rasmus I spoke to the office gift master and he suggested a change to my worklfow which will avoid these issues from my side in the future. Will do an update tomorrow.

**Comment by JC008** [comment]
I did a `paket update` and it fixed the dependencies issues. But I see this an warning re [ElasticSearch](https://ci.appveyor.com/project/rasmusnu/eventflow/build/0.36.2293#L178)  and all my test except elastic search is passing.

**Comment by rasmus** [comment]
@JC008 The tests are failing to do this

`Could not load file or assembly 'Newtonsoft.Json, Version=9.0.0.0, Culture=neutral`

**Comment by JC008** [comment]
Sorry - I scanned for `red` markers only on the last build

File: RELEASE_NOTES.md [hunk 1]
```diff
@@ -8,6 +8,17 @@
   interface allows control of how EventFlows runs tasks. Please note that
   EventFlow will only use `ITaskRunner` in very limited cases, e.g. if
   there's implantations of `ISubscribeAsynchronousTo<,,>`
+* New: `EventFlow.SQLite` now uses an `ISQLiteDatabaseMigrator` implementation 
+  to manage the creation of `EventFlow` SQL schemas when using SQLite. The 
+  behaviour and implementation is now on par with `EventFlow.MsSql` and you can 
+  use embedded resource `SQLite` scripts and the `ISQLiteDatabaseMigrator` 
+  implemenation to generate database schemas for `SQLite` read models too.
+* New: The following Nuget dependencies have been added
+  - `Helpz.SQLite` v0.2.64
```
Comment (rasmus) [inline RELEASE_NOTES.md]: @JC008 Remove release notes related to updated test frameworks. The release notes are put into nuget.org descriptions and people can't download any packages related to tests

File: RELEASE_NOTES.md [hunk 2]
```diff
@@ -8,6 +8,17 @@
...
+* Breaking: The following Nuget dependencies have been updated
+  - `dbup-sqlite` v3.3.5 (up from v3.3.4)
+  - `Helpz` v0.2.64 (up from v0.1.8)
```
Comment (rasmus) [inline RELEASE_NOTES.md]: @JC008 Ditto here, remove `Helpz`

File: Source/SolutionInfo.cs [hunk 3]
```diff
@@ -6,8 +6,8 @@
 using System.Reflection;
 
 [assembly: AssemblyDescription("developer build")]
-[assembly: AssemblyVersion("0.0.1")]
-[assembly: AssemblyFileVersion("0.0.1")]
-[assembly: AssemblyInformationalVersion("0.0.1")]
+[assembly: AssemblyVersion("0.33.2172")]
+[assembly: AssemblyFileVersion("0.33.2172")]
```
Comment (rasmus) [inline Source/SolutionInfo.cs]: @JC008 Revert these changes, leave `0.0.1` in the files

File: RELEASE_NOTES.md [hunk 4]
```diff
@@ -8,6 +8,17 @@
...
+  - `System.Data.SQLite.Core` v1.0.103 (up from v1.0.102)
```
Comment (rasmus) [inline RELEASE_NOTES.md]: @JC008 You updated `Newtonsoft.Json` to v9.0 as well, put that in

**Comment by rasmus** [comment]
@JC008 A few comments, but it look very nice 👍

**Comment by rasmus** [comment]
Closing this, three years behind and not updated
